test/unit/helper/labelGenerator_default.js

```diff
@@ -175,14 +175,14 @@ module.exports.tests.default_country = function(test, common) {
     t.end();
   });
 
-  test('country', function(t) {
+  test('country layer labels should only use the `country` field and not the `name`', function(t) {
     var doc = {
-      'name': 'country name',
+      'name': 'source country name',
       'layer': 'country',
       'country_a': 'country code',
-      'country': 'country name'
+      'country': 'hierarchy country name'
     };
-    t.equal(generator(doc),'country name');
+    t.equal(generator(doc),'hierarchy country name');
```

Comment:
We could use the `name` field but that would generate a label that we wouldn't be able to search for again.  That is, geonames' name for `China` is `The People's Republic of China`.  While libpostal understands it to be a country, we won't find anything since we only import the WOF name `China`.